en/get-started.md

```diff
@@ -317,6 +317,14 @@ Before proceeding, make sure the following requirements are satisfied:
             helm install --namespace tidb-admin --name tidb-operator pingcap/tidb-operator --version v1.1.0
             ```
 
+            For Chinese mainland users, you may use local images:
```

Comment:
@ran-huang  Please help confirm if the words `Chinese mainland` is suitable in the docs.

Reply:
@cofyc It's not necessary to write special note for mainland users in English docs, since mainland users barely read English docs.

However, we do recommend that you add this note in the **Chinese version**, but a little change of wording is required. 
Here's my suggestion: please **don't use `country/region`** to distinguish users, which could be **sensitive** in some circumstances. We should just give two different images (docker & Alicloud), and allow Chinese users to choose freely.

Reply:
We need to add this note to the English version as our colleagues may read the English version.

How about `If the network connection to the Docker Hub is slow, you can try images hosted in Alibaba Cloud`?

Reply:
Nice input. We can go with this one.

Reply:
Updated